apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -409,6 +411,14 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
             f'"size" attribute on "{name}" should only be set if display is "dropdown".'
         )
 
+    if (
+        pl.has_attrib(element, "placeholder")
+        and get_display_type(element) is not DisplayType.DROPDOWN
```

Comment:
Can you wrap this with the above in a single if statement? Like
```python
if display is not dropdown:
    if has_attrib(size):
        raise exception
    if has_attrib(placeholder):
        raise exception
```
keeping the dropdown only attributes grouped together.